Entities/person.rb

```diff
@@ -2,16 +2,19 @@
 require_relative '../Decorate/capitalize_decorator'
 require_relative '../Decorate/trimmer_decorator'
 require_relative '../Decorate/base_decorator'
+require_relative '../Associations/rental'
+
 class Person < Nameable
-  attr_reader :id
+  attr_reader :id, :rentals
   attr_accessor :name, :age
```

Comment:
- [ ] rentals should be included in attr_accessor.KIndly add it.

Reply:
Done